core/vm/interpreter.go

```diff
@@ -46,7 +46,6 @@ type ScopeContext struct {
 // EVMInterpreter represents an EVM interpreter
 type EVMInterpreter struct {
 	evm *EVM
-	cfg Config
```

Comment:
IMHO it would be better to store the `JumpTable` here. The table is misplaced in `Config` anyway, it is impossible to set it to a meaningful value outside of core/vm.

Reply:
I don't want to rewrite any logic but only move the config so that changing it has an effect and the EVM struct can be reused